applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_potential_flow_utilities.cpp

```diff
@@ -612,7 +612,12 @@ KRATOS_TEST_CASE_IN_SUITE(ComputeUpwindedDensityDerivativeWRTVelocitySquaredSupe
     const double density_derivative_accel = PotentialFlowUtilities::ComputeUpwindedDensityDerivativeWRTVelocitySquaredSupersonicAccelerating<2,3>(
         current_velocity, local_mach_number_squared, upwind_mach_number_squared, model_part.GetProcessInfo());
 
-    KRATOS_CHECK_RELATIVE_NEAR(density_derivative_accel, 6.3365379876067950e-07, 1e-15);
+    const double reference = 6.33653798760679499e-07;
+    std::cout.precision(32);
+    KRATOS_WATCH(density_derivative_accel)
+    KRATOS_WATCH(reference)
```

Comment:
I think you unintentionally pushed this.

Reply:
I wanted to check if I can see where the problem comes from in the CI, removed in https://github.com/KratosMultiphysics/Kratos/pull/8456/commits/3e25095734bbbb157fa76a9d64d209ad128ec138